### PR TITLE
spring-backend

### PR DESCRIPTION
**Pull request by ManaliWani**
This code has backend logic to store users in mongodb database and restful api's to do the same.

### CODE REVIEW

File: SansactionScreeningBackend/target/classes/META-INF/maven/com.example/SansactionScreeningBackend/pom.properties [hunk 1]
```diff
@@ -0,0 +1,7 @@
+#Generated by Maven Integration for Eclipse
+#Sun Mar 31 14:26:47 IST 2019
+m2e.projectLocation=D\:\\SpringWorkspace\\SansactionScreeningBackend
```
Comment (ajinkya007) [inline SansactionScreeningBackend/target/classes/META-INF/maven/com.example/SansactionScreeningBackend/pom.properties]: CR+LF is EOL char here.

**Comment by ajinkya007** [comment]
Please don't commit the class files. Remove those in the next commit.

**Comment by ajinkya007** [comment]
Why do we have two pom.xml's ?

**Comment by ajinkya007** [comment]
Also please ensure you have the package and java classes named as per the java standard convention

**Comment by ManaliWani** [comment]
The pom.xml in target folder is auto generated... I don't know how it got
generated.... But maybe because I have used spring Initializer to create
the project.

**Comment by ManaliWani** [comment]
com.example shouldn't be used???

**Comment by ManaliWani** [comment]
How can we solve this (CR LF).... Is it a problem..??

And that pom.properties file in target folder is also auto generated

**Pull request merged**